sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyProperties.java

```diff
@@ -109,17 +109,17 @@ public String name() {
      *
      * @return the enabled value
      */
-    public Boolean enabled() {
+    public Boolean isEnabled() {
         return this.enabled;
     }
 
     /**
      * Set the enabled value.
      *
      * @param enabled The enabled value to set
-     * @return the KeyBase object itself.
+     * @return the KeyProperties object itself.
```

Comment:
@return the **updated** KeyProperties object.

Reply:
fixed.